javascript/package-lock 2.json

```diff
@@ -0,0 +1,14713 @@
+{
```

Comment:
This file should  not be checked into the code base.  Not sure why it exists.  Please remove